pkg/kubectl/cmd/create/create_job.go

```diff
@@ -89,15 +91,30 @@ func NewCmdCreateJob(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddDryRunFlag(cmd)
 	cmd.Flags().String("from", "", "The name of the resource to create a Job from (only cronjob is supported).")
+	cmd.MarkFlagRequired("from")
 
 	return cmd
 }
 
+func (o *CreateJobOptions) Validate() (err error) {
+	if o.Name == "" {
+		return fmt.Errorf("name must be specified")
+	}
+	if o.From == "" {
```

Comment:
No need to add this check, since this flag has been marked as required. Cobra will do this check.